delta/app/src/main/resources/logback.xml

```diff
@@ -25,6 +25,9 @@
         <appender-ref ref="STDOUT" />
     </logger>
 
+    <logger name="slick" level="INFO"/>
```

Comment:
To remove ? These aren't too verbose ?

Reply:
They are verbose. I couldn't make sense of what was going on in DEBUG mode

Reply:
So we could have warn or error by default when we are not in dev mode ?

Reply:
We can change that in other PRs as we desired. This PR does not concern logging that much.